source/org/zfin/framework/dao/VocabularyDAO.java

```diff
@@ -0,0 +1 @@
+package org.zfin.framework.dao;

import org.zfin.framework.Vocabulary;

public class VocabularyDAO extends BaseSQLDAO<Vocabulary> {

    public VocabularyDAO() {
        super(Vocabulary.class);
    }

}

```

Comment:
i'm seeing weird line-endings here.

Reply:
I just created a PR to this PR for fixing the line-endings

Reply:
Thanks.